cms/settings/dev.py

```diff
@@ -86,6 +86,8 @@
     WAGTAILADMIN_LOGIN_URL = env.get("WAGTAILADMIN_LOGIN_URL", f"{AUTH_STUB_URL}/florence/login")
     LOGOUT_REDIRECT_URL = env.get("LOGOUT_REDIRECT_URL", f"{AUTH_STUB_URL}/florence/logout")
     AWS_REGION = "eu-west-2"
+else:
+    SESSION_COOKIE_AGE = 60 * 60 * 24  # 1 day in seconds
```

Comment:
Question: Should this just live in `base.py` instead? Given we're already manually applying it as an env var anyway.